extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/FunctionPrivilegesBuilderSuite.scala

```diff
@@ -104,12 +108,127 @@ abstract class FunctionPrivilegesBuilderSuite extends AnyFunSuite
     spark.stop()
     super.afterAll()
   }
+
+  val HIVE_MASK_HASH_CLASS = "org.apache.hadoop.hive.ql.udf.generic.GenericUDFMaskHash"
+  val HIVE_FUNCTION_NAME_BASE = "hive_func"
+  val OBJECT_NAME_SEPARATOR = "_"
+
+  protected def checkFunctions(sql: String, count: Int, spark: SparkSession): Unit = {
+    val funcNames = ArrayBuffer.empty[String]
+    for (index <- 0.until(count)) {
+      funcNames.append(s"${if (index % 2 == 0) reusedDb else reusedDb2}" +
+        s".$HIVE_FUNCTION_NAME_BASE$OBJECT_NAME_SEPARATOR$index")
```

Comment:
Suggested to use placeholder brackets for better readability.
=> s".${HIVE_FUNCTION_NAME_BASE}${OBJECT_NAME_SEPARATOR}$index"